src/runtime/entitlements-manager-test.js

```diff
@@ -131,6 +131,10 @@ describes.realWin('EntitlementsManager', {}, env => {
           .withExactArgs('toast')
           .returns(Promise.resolve(null))
           .atLeast(0);
+      storageMock.expects('get')
+          .withExactArgs('isreadytopay')
+          .returns(Promise.resolve('undefined'))
```

Comment:
Why string here? I'd think `undefined` as a string is a mistake resulting from incorrect write. Otherwise, the storage would return `null` for a missing value.